### PR TITLE
BXMSPROD-535: updating git repository caches

### PR DESCRIPTION
**Pull request by akoufoudakis**
Removing the cache repositories. Temporarily fix.

### CODE REVIEW

File: jenkins-slaves/ansible/roles/kie/tasks/main.yml [hunk 1]
```diff
@@ -153,49 +153,6 @@
     path: /home/jenkins/git-repos
```
Comment (almope) [inline jenkins-slaves/ansible/roles/kie/tasks/main.yml]: Do we need still the directory to be created?

Comment (akoufoudakis) [reply jenkins-slaves/ansible/roles/kie/tasks/main.yml]: Oh. It slipped through. I removed the Jenkins permissions for git-repos, but missed that one. Correcting right now.

**Comment by mareknovotny** [comment]
there is some misunderstanding here with what the jira requests
we should not remove git cache repos, but update them to be in sync with repository-list.txt file list

**Comment by akoufoudakis** [comment]
As discussed, repos need to be updated, but not the removed.

File: jenkins-slaves/ansible/roles/kie/tasks/main.yml [hunk 2]
```diff
@@ -158,34 +157,30 @@
   with_items:
    - kiegroup/lienzo-core
    - kiegroup/lienzo-tests
-   - errai/errai
```
Comment (mareknovotny) [inline jenkins-slaves/ansible/roles/kie/tasks/main.yml]: this one I would keep until we remove Errai from Upstream builders

File: jenkins-slaves/ansible/roles/kie/tasks/main.yml [hunk 3]
```diff
@@ -158,34 +157,30 @@
...
+   - kiegroup/droolsjbpm-build-bootstrap
    - kiegroup/kie-soup
    - kiegroup/appformer
-   - kiegroup/droolsjbpm-build-bootstrap
    - kiegroup/droolsjbpm-knowledge
    - kiegroup/drools
    - kiegroup/optaplanner
-   - kiegroup/optaweb-employee-rostering
    - kiegroup/jbpm
-   - kiegroup/jbpm-work-items
```
Comment (mareknovotny) [inline jenkins-slaves/ansible/roles/kie/tasks/main.yml]: this should not be removed IMHO, see https://github.com/kiegroup/droolsjbpm-build-bootstrap/blob/master/script/repository-list.txt#L20

File: jenkins-slaves/ansible/roles/kie/tasks/main.yml [hunk 4]
```diff
@@ -204,4 +199,5 @@
     owner: jenkins
     group: jenkins
     mode: 0755
-...
\ No newline at end of file
+...
+...
```
Comment (mareknovotny) [inline jenkins-slaves/ansible/roles/kie/tasks/main.yml]: doubled 3 dots? Is this intentional?

**Pull request merged**